test/pummel/test-regress-GH-814.js

```diff
@@ -46,7 +46,7 @@ var timeToQuit = Date.now() + 8e3; //Test during no more than this seconds.
     }
   }
   else {
-    throw Error("Buffer GC'ed test -> FAIL");
+    throw new Error("Buffer GC'ed test -> FAIL");
```

Comment:
The lint rules allow for double quotes if it is to avoid escaping a single quote as it is here. I'm inclined to leave it.